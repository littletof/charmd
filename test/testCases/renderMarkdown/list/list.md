+ Create a list by starting a line with `+`, `-`, or `*`
  - Indented list item:
    * Subsublist item
      + Subsublist item
+ Main list item

1. Lorem ipsum dolor sit amet
2. Consectetur adipiscing elit
3. Integer molestie lorem at massa

- [x] Some
- [ ] Other