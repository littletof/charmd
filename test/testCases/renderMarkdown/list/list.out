[90m- [39mCreate a list by starting a line with [7m + [27m, [7m - [27m, or [7m * [27m
  [90mâ—¦ [39mIndented list item:
    [90mâ–ª [39mSubsublist item
      [90mâ–¸ [39mSubsublist item
[90m- [39mMain list item

[90m1. [39mLorem ipsum dolor sit amet
[90m2. [39mConsectetur adipiscing elit
[90m3. [39mInteger molestie lorem at massa

[90m- [39m[x] Some
[90m- [39m[ ] Other
